#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_news.texi.dpatch by Christian Marillat <marillat@debian.org>
##
## All lines beginning with ## DP:' are a description of the patch.
## DP:

@DPATCH@

--- librep-0.17.orig/man/news.texi	Wed Jul 18 15:41:34 2001
+++ librep-0.17/man/news.texi	Wed Jul 18 15:42:04 2001
@@ -35,7 +35,7 @@
 @item New functions in @code{rep.data} module, @code{list->vector} and
 @code{vector->list}
 
-@item New macro @code{define-special-form}. A combination of
+@item New macro @code{define-special-variable}. A combination of
 @code{defvar} and @code{setq} -- it always makes the variable special
 and it always sets it to the given value
 
